pkg/build/registry/buildconfiginstantiate/rest.go

```diff
@@ -178,7 +178,31 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 	}
 	remaining := h.r.Timeout - time.Now().Sub(start)
 
+	// Attempt to cancel the build if it did not start running
```

Comment:
any suggestions on how to create one that doesn't have to wait 5 minutes to timeout?

since the timeout isn't configurable and can't easily be made configurable from the client side.